be/src/runtime/hdfs/hdfs_fs_cache.cpp

```diff
@@ -21,9 +21,9 @@
 
 namespace starrocks {
 
-// Try to get azure cloud properties from FSOptions, if azure credentials not existed, return nullptr.
-// TODO(SmithCruise): Should remove when using azure cpp sdk
-static const std::vector<TCloudProperty>* get_azure_cloud_properties(const FSOptions& options) {
+// Try to get cloud properties from FSOptions, if cloud configuration not existed, return nullptr.
```

Comment:
For GCS, we use gcs sdk in FE, but hdfs sdk in BE?

Reply:
gcs sdk in one of the hdfs sdk. I will change the comment later.